man/networkd.conf.xml

```diff
@@ -61,6 +61,13 @@
         <listitem><para>Specifies the time interval to calculate the traffic speed of each interface.
         If <varname>SpeedMeter=no</varname>, the value is ignored. Defaults to 10sec.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>DropForeignRoutes=</varname></term>
```

Comment:
s/Drop/Ignore/ would be more appropriate, if I understand this patch correctly

Reply:
Ignore is very nice